### ISSUE TITLE
Plot Additional Data and Make Scrollable

### ISSUE THREAD

**Issue by channemann** [loudnate/openaps-monitor#19]
Would be excellent to be able to plot 12 - 24 hours and make it scrollable. The monitor may not be the best place for this, but making the live view also the historical / analysis view would be very cool.

**Comment by loudnate** [comment]
Yeah there's a few pieces here. First, we need to change the code that takes system time and ensures the horizontal axis includes it. We should replace current time with a vertical line or something, and update it on a regular interval.
Second, we should decide on the maximum time to show in the viewport, something pixel-based so smaller screens trigger scrolling. The amount of data is currently based on the pump history, glucose history, and prediction glucose provided.

**Comment by channemann** [comment]
Would we need to change the iter_glucose_hours and iter_pump_hours values in the loop, or simply append the new data each time?

**Comment by channemann** [comment]
If we're moving to Highcharts ...

http://www.highcharts.com/docs/chart-concepts/understanding-highstock

**Comment by loudnate** [comment]
Openaps only supports overwriting files, so you'd need to just increase those values.
